src/Appwrite/Extend/Exception.php

```diff
@@ -116,9 +116,16 @@ class Exception extends \Exception
     public const STORAGE_INVALID_CONTENT_RANGE     = 'storage_invalid_content_range';
     public const STORAGE_INVALID_RANGE             = 'storage_invalid_range';
 
+    /** VCS */
+    public const INSTALLATION_NOT_FOUND              = 'installation_not_found';
+    public const PROVIDER_REPOSITORY_NOT_FOUND       = 'provider_repository_not_found';
+    public const REPOSITORY_NOT_FOUND                = 'repository_not_found';
```

Comment:
whats the difference between these two ?

Reply:
https://github.com/appwrite/appwrite/pull/5725#discussion_r1291125308

Explained here